src/create-support-ticket.ts

```diff
@@ -189,13 +189,8 @@ export const handler = async (
     console.log(`finished processing event with ID: ${eventIdentifier}`);
     return input;
   } catch (error) {
-    console.error(
-      `[Error occurred], unable to send suspicious activity event with ID: ${eventIdentifier} to Zendesk, ${
-        (error as Error).message
```

Comment:
Found this console.error useful as part of a recent debugging activity. Why is it removed pls?

Reply:
Can we have a quick run through of the changes in this PR pls when you are free